runtime/gc_glue_java/ObjectModel.hpp

```diff
@@ -1,6 +1,6 @@
 
 /*******************************************************************************
```

Comment:
There are some extra classes added to this PR now.

Reply:
Fixed the extraneous files.